### PR TITLE
Inherited profiles do not work when a property is overridden in the child POM

### PR DESCRIPTION
**Pull request by jglick**
Sorry @andresrc this trick just does not work. You really need to keep the profiles in your local settings. Perhaps @stephenc knows of some trick to make it work, but according to [this](http://stackoverflow.com/q/18641504/12916) it is not possible.

@reviewbybees

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,3 +38,92 @@ In order to use the new POM:
...
+
+It is handy to be able to select different Jenkins baselines with a Maven profile.
+To set this up, you must edit your `~/.m2/settings.xml` to include some new entries in the `<profiles>` section.
+For example:
+
+```xml
+<profile>
+    <id>jenkins-2</id>
+    <properties>
+        <jenkins.version>2.0</jenkins.version>
```
Comment (jglick) [inline README.md]: Not an LTS, but handy for testing.

File: README.md [hunk 2]
```diff
@@ -38,3 +38,92 @@ In order to use the new POM:
...
+        <hpi-plugin.version>1.115</hpi-plugin.version>
+        <stapler-plugin.version>1.17</stapler-plugin.version>
+        <java.level>7</java.level>
+    </properties>
+</profile>
+<profile>
+    <id>jenkins-651</id>
+    <properties>
+        <jenkins.version>1.651.1</jenkins.version>
```
Comment (jglick) [inline README.md]: This one was missing.

File: README.md [hunk 3]
```diff
@@ -38,3 +38,92 @@ In order to use the new POM:
...
+        <hpi-plugin.version>1.115</hpi-plugin.version>
+        <stapler-plugin.version>1.17</stapler-plugin.version>
+        <java.level>7</java.level>
+    </properties>
+</profile>
+<profile>
+    <id>jenkins-642</id>
+    <properties>
+        <jenkins.version>1.642.3</jenkins.version>
```
Comment (jglick) [inline README.md]: This was still at 1.642.1.

**Comment by andresrc** [comment]
ups! :bee:

**Comment by amuniz** [comment]
:bee:

**Pull request merged**